styles/base.less

```diff
@@ -112,7 +112,7 @@ atom-text-editor.is-focused, :host {
 }
 
 .storage {
-  color: @base08;
+  color: @base14;
```

Comment:
Can you explain me why you change it? I am not adding this change , this change every storage colors.